utils/env_preferences/Cargo.toml

```diff
@@ -30,8 +32,14 @@ features = [
     "Globalization",
     "Globalization_DateTimeFormatting",
     "Win32",
-    "Win32_Globalization"
+    "Win32_Globalization",
 ]
 
 [dev-dependencies]
-icu_locale = { path = "../../components/locale" }
+icu_locale = { workspace = true }
+windows-core = "0.60.1"
+
+[features]
```

Comment:
I'm not sure we need features for this. They don't pull in any additional dependencies, and the crate is pretty useless with them turned off. `cfg`ing by target should be enough